static/bidder-info/kargo.yaml

```diff
@@ -10,6 +10,9 @@ capabilities:
       - video
       - native
 userSync:
+  iframe:
+    url: "https://crb.kargo.com/api/v1/initsyncpbs?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&r={{.RedirectURL}}"
+    userMacro: "$UID"
```

Comment:
The user sync url is not setting UID for the following URL. Please check - 
```
https://crb.kargo.com/api/v1/initsyncpbs?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&r=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dkargo%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24UID
```

Reply:
Same is happening for me

Reply:
If youve only called once it wont work until we've had a chance to place a cookie. Refresh and you should see it. 
![Screenshot 2024-08-14 at 11 08 59 AM](https://github.com/user-attachments/assets/7f160a43-571c-4630-8bcc-9bd1a1f3c68f)

I'm addressing that extra `\n` now but that shouldnt cause any issues.

Reply:
Please raise this PR once you have implemented a loop-redirect to place a cookie on the first attempt. We should be able to merge it when that is resolved.